web/src/main.js

```diff
@@ -1133,9 +1133,12 @@ function renderHistory(state) {
     .render();
 
 
-  zoneDetailsTimeSlider.onChange((selectedZoneTimeIndex) => {
-    if (getState().application.selectedZoneTimeIndex !== selectedZoneTimeIndex) {
-      dispatch({type: 'UPDATE_SLIDER_SELECTED_ZONE_TIME', payload: {selectedZoneTimeIndex: selectedZoneTimeIndex}})
+  zoneDetailsTimeSlider.onChange((selectedZoneTimeIndexInput) => {
+    if (getState().application.selectedZoneTimeIndex !== selectedZoneTimeIndexInput) {
+      const selectedZoneTimeIndex = selectedZoneTimeIndexInput === (history.length - 1)
```

Comment:
can you add a small comment that states that if the slider is on the last value, then we set the time to null to effectively use the current state?

Reply:
Done !